main.tf

```diff
@@ -1,13 +1,13 @@
 locals {
-  deny_leaving_orgs_effect             = var.deny_leaving_orgs ? "Deny" : "Allow"
-  deny_creating_iam_users_effect       = var.deny_creating_iam_users ? "Deny" : "Allow"
-  deny_deleting_kms_keys_effect        = var.deny_deleting_kms_keys ? "Deny" : "Allow"
-  deny_deleting_route53_zones_effect   = var.deny_deleting_route53_zones ? "Deny" : "Allow"
-  deny_deleting_cloudwatch_logs_effect = var.deny_deleting_cloudwatch_logs ? "Deny" : "Allow"
-  deny_root_account_effect             = var.deny_root_account ? ["Deny"] : []
-  protect_s3_buckets_effect            = var.protect_s3_buckets ? ["Deny"] : []
-  protect_iam_roles_effect             = var.protect_iam_roles ? ["Deny"] : []
-  limit_regions_effect                 = var.limit_regions ? ["Deny"] : []
+  deny_leaving_orgs_statement             = var.deny_leaving_orgs ? [""] : []
+  deny_creating_iam_users_statement       = var.deny_creating_iam_users ? [""] : []
+  deny_deleting_kms_keys_statement        = var.deny_deleting_kms_keys ? [""] : []
+  deny_deleting_route53_zones_statement   = var.deny_deleting_route53_zones ? [""] : []
+  deny_deleting_cloudwatch_logs_statement = var.deny_deleting_cloudwatch_logs ? [""] : []
+  deny_root_account_statement             = var.deny_root_account ? [""] : []
+  protect_s3_buckets_statement            = var.protect_s3_buckets ? [""] : []
+  protect_iam_roles_statement             = var.protect_iam_roles ? [""] : []
+  limit_regions_statement                 = var.limit_regions ? [""] : []
 }
```

Comment:
Please add a comment explaining how this works with the `dynamic` directives so this is a little more clear.

Reply:
I did this but it might be a bit wordy. Lmk if you see an obvious way to pare it down.